app/lib/actions/remote_execution/run_hosts_job.rb

```diff
@@ -38,6 +41,10 @@ def create_sub_plans
         end
       end
 
+      def finalize
+        input.delete(:secrets)
```

Comment:
Is this still needed, with `HideSecrets` in place?

Reply:
The `HideSecrets` middleware only prevents showing the secrets in the webui. Anyone with access to the database could still look there to find the secrets. For this reason we should delete them by hand when they're not needed anymore

Reply:
Oh, I"ve seen https://github.com/theforeman/foreman-tasks/pull/297/files#diff-03dde90ec7e9caa322015dee15f4e956R103 also doing some wiping, but now I see it is run_host_job vs. run_hosts_job. Nm then